eva-submission-docker/eva_docker/Dockerfile

```diff
@@ -0,0 +1,81 @@
+FROM ubuntu:22.04
+
+ENV JAVA_VERSION 8
+ENV NXF_VER 21.10.5
+ENV VCF_VALIDATOR_VERSION 0.9.4
+ENV MONGO_VERSION 4.4
+ENV EVA_ACCESSION_VERSION v0.6.21
+ENV EVA_PIPELINE_VERSION v2.9.1
+ENV EVA_SUBMISSION_VERSION v1.10.6
```

Comment:
I think pinning these is fine for now but might have to change depending on how we use this for CI/CD in the future - e.g. do we want to make sure we're always using the most recent tagged versions, etc.

Reply:
Have left these as it is for now. I guess we can decide later based on we are going to use docker image

Reply:
We should separate the tools we developed (likely to change often) and the one that change more rarely like java, nextflow, mongo. I suggest a separated ticket to manage this.